resources/views/store/printables/partials/masthead.blade.php

```diff
@@ -0,0 +1,7 @@
+<div class="header">
+	<img src="{{ asset('store/assets/logo.png') }}">
+	<div class="print-notes">
+		<p>Please do not photocopy blanks of this form to reuse.</p>
+		<p>{{ $specificPrintNote }}</p>
```

Comment:
Traditionally we've used _ not caMel for blade templates / php variables - but not bothered.